robottelo/constants.py

```diff
@@ -83,6 +84,34 @@
 GCE_NETWORK_DEFAULT = 'default'
 GCE_EXTERNAL_IP_DEFAULT = True
 
+# AzureRM specific constants
+AZURERM_VALID_REGIONS = {
+    'eastasia': 'East Asia', 'southeastasia': 'Southeast Asia',
+    'centralus': 'Central US', 'eastus': 'East US', 'eastus2': 'East US 2', 'westus': 'West US',
+    'northcentralus': 'North Central US', 'southcentralus': 'South Central US',
+    'northeurope': 'North Europe', 'westeurope': 'West Europe', 'japanwest': 'Japan West',
+    'japaneast': 'Japan East', 'brazilsouth': 'Brazil South', 'australiaeast': 'Australia East',
+    'australiasoutheast': 'Australia Southeast', 'southindia': 'South India',
+    'centralindia': 'Central India', 'westindia': 'West India', 'canadacentral': 'Canada Central',
+    'canadaeast': 'Canada East', 'uksouth': 'UK South', 'ukwest': 'UK West',
+    'westcentralus': 'West Central US', 'westus2': 'West US 2', 'koreacentral': 'Korea Central',
+    'koreasouth': 'Korea South', 'francecentral': 'France Central', 'francesouth': 'France South',
+    'australiacentral': 'Australia Central', 'australiacentral2': 'Australia Central 2',
+    'uaecentral': 'UAE Central', 'uaenorth': 'UAE North', 'southafricanorth': 'South Africa North',
+    'southafricawest': 'South Africa West', 'switzerlandnorth': 'Switzerland North',
+    'switzerlandwest': 'Switzerland West', 'germanynorth': 'Germany North',
+    'germanywestcentral': 'Germany West Central', 'norwaywest': 'Norway West',
+    'norwayeast': 'Norway East'}
+AZURERM_RHEL7_FT_IMG_URN = 'RedHat:RHEL:7-RAW:latest'
+AZURERM_RHEL7_UD_IMG_URN = 'RedHat:RHEL:7-RAW-CI:7.6.2019072418'
+AZURERM_RG_DEFAULT = 'SATQE'
+AZURERM_PLATFORM_DEFAULT = 'Linux'
+AZURERM_VM_SIZE_DEFAULT = 'Standard_B2ms'
+AZURERM_PREMIUM_OS_Disk = True
+AZURERM_FILE_URI = 'https://gist.githubusercontent.com/' \
+                   'vijay8451/72150405d99e0711056e50151251e2b1/raw/' \
+                   '734316fd56578e0b5777e686fb4119733ea457d9/check.sh'
```

Comment:
Try to avoid backslash unless they are absolutely necessary:
```python
AZURERM_FILE_URI = ('https://gist.githubusercontent.com/'
                    'vijay8451/72150405d99e0711056e50151251e2b1/raw/'
                    '734316fd56578e0b5777e686fb4119733ea457d9/check.sh')
```

Also - is there a chance to not use private gist here? Perhaps introduce new file into repo that will be uploaded?

Reply:
Good observation :+1:  .. added it.